### PR TITLE
SPM-915: Added "package_versions" endpoint.

### PR DESCRIPTION
**Pull request by Josca**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: manager/controllers/package_versions.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+// @ID packageVersions
+// @Security RhIdentity
+// @Accept   json
+// @Produce  json
+// @Param    limit          query   int     false   "Limit for paging, set -1 to return all"
+// @Param    offset         query   int     false   "Offset for paging"
+// @Param    package_name    path    string    true  "Package name"
+// @Param    tags            query   []string  false "Tag filter"
+// @Param    filter[system_profile][sap_system]   query  string  false "Filter only SAP systems"
+// @Param    filter[system_profile][sap_sids][in] query []string  false "Filter systems by their SAP SIDs"
```
Comment (semtexzv) [inline manager/controllers/package_versions.go:59]: Do we really need to have this filtering level here ?

**Review by semtexzv — APPROVED** [review]
One question, otherwise looks good

File: manager/controllers/package_versions.go (hunk 1 above)
Comment (Josca) [reply manager/controllers/package_versions.go:59]: I think we need the same filtering level as used for package_systems to show relevant versions when filters are applied.

Comment (Josca) [reply manager/controllers/package_versions.go]: Ok, removing filters and tags support here. I will that in a separate commit so we can easily add it back if we will need that.

**Pull request merged**